content/projects/gsoc/mentors.adoc

```diff
@@ -47,18 +45,281 @@ Mentors are expected to...
+** of course we appreciate if you help us find more mentors if you can
+* participate in the weekly public meeting
+* make sure the students follow the process and that their application meets the requirements in the link:https://docs.google.com/document/d/1dIlPLXfLbFsvcaHFuwmH9_lSCVm9m6-SgNYTNAnSZpY/edit[template]
+** make sure the students determines whether they are link:gsoc/students/#eligibility-steps[eligible]
+* if the student proposes a genuine project idea in your area of interest or expertise, make sure it is presented and discussed in the community
+
+You can still submit new project ideas during this phase.
+
+This is a very important phase, use it to get to know the students who apply to projects that are of interest to you.
+
+For this deadline, please see the
+link:https://developers.google.com/open-source/gsoc/timeline[timeline]).
+
+=== During student slot request and final selection phase
+
+*This phase lasts 3 weeks.*
+
+*Expectation:* about 5 hours total.
+
+This phase is divided in two sub-phases:
+
+* slot request phase (2 weeks)
+* final project selection phase (1 week)
+
+Mentor team are formed during this phase, student proposals are ranked for slot requests, and a final selection is made.
+
+Note that *we are NOT allowed to communicate any information regarding the selection to students during this phase until Google makes the official Student Project Announcements.*
+We do not make this announcement, Google does.
+
+==== Regarding forming mentor teams
+
+In the Jenkins Org, we have the concepts of lead mentor and co-mentors.
+
+The lead mentor has the responsibilities of a co-mentor, and is also primarily responsible for the administrative duties
+(fill in the evaluation documents,
+ensuring representation at meetings,
+and report issues to the Org Admin team).
+
+The co-mentors work with the student during all phases on the program,
+answering the student questions, coaching, advising, motivating, unblocking, reviewing code and pull-requests, ensuring the process is followed,
+ensuring communications are in public as this is an open source program, and report issues to the lead mentor, etc.
+
+Subject matter experts and active community contributors also participate in GSoC but not in official mentor roles.
+
+==== Regarding the slot request sub-phase
+
+*This sub-phase lasts 2 weeks.*
+
+The goal of this phase is to determine and request the minimum and the maximum number of projects we can take on as an organization.
+This process is explained in the Mentor Guide in the
+link:https://google.github.io/gsocguides/mentor/selecting-students-and-mentors#slot-count[Slot Count] section.
+
+We have two weeks to send in our slot request to Google.
+*It is an intense and critical period for Org Admins and mentors*,
+as this determines who participates in the rest of the program!
+Note that this phase can coincide with the celebrations around Easter.
```

Comment:
I suggest removing the specific holiday mention, because we do not bother mentioning holidays in other countries/cultures. A generic reference to holidays is fine, but I do not think specific is helpful here

Reply:
Done